Runtime/RosSharpDefinitions/Robot.cs

```diff
@@ -92,6 +94,14 @@ private List<Plugin> ReadPlugins(XElement node)
             return plugins.ToList();
         }
 
+        private List<Tuple<string,string>> ReadDisableCollision(XElement node)
+        {
+            var disable_collisions =
```

Comment:
nit: I think we should use less "var" implicit types. we could use `System.Collections.Generic.IEnumerable<Tuple<string, string>>` as the explicit type

Reply:
This was in line with code written previously. This code was part of ROS #